src/main/java/www/purple/mixxy/helpers/FacebookAuthHelper.java

```diff
@@ -0,0 +1,85 @@
+package www.purple.mixxy.helpers;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.UnsupportedEncodingException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLConnection;
+import java.net.URLEncoder;
+
+import com.google.api.client.http.HttpTransport;
+import com.google.api.client.http.javanet.NetHttpTransport;
+
+public class FacebookAuthHelper {
+	public static final String FB_APP_ID = "1525243031118871";
+	public static final String FB_APP_SECRET = "33448c710b2ba227461752c3abe7dc06";
```

Comment:
Whoa whoa whoa, I just realized. this needs to be secret.  It's out in the open.  We gotta regenerate the secret and try again, this simply won't do.